5-entity-strategic-council.rst

```diff
@@ -0,0 +1,154 @@
+* Third party audits of financial behavior enforce good behavior and create transparency.
+* Bad actors should be able to be removed from any decision making body for egregious violations of trust or misbehavior.
+
+*4. Who executes?*
+
+* Anyone should be able to participate in the execution.
+* Over time the best executors should have reputation accrue and be able to receive more funds.
+* There is value in having distributed executors across geographies and across entities
+* There is value in identifying Executors who have long term commitments to Zcash and will be available for long term support and maintenance of their work
+
+5. How are executors held accountable to excellent execution?
+
+* Excellent execution comes from having verifiable hypotheses, backed up with data, and clear milestones.
+* Executors need to submit concrete plans, with clear goals and metrics, and be judged according to both whether or not the goals were reasonable and whether they accomplished those goals (ideally in a measurable way using metrics).
+* Execution is best measured by pre-defining success and failure criteria, prior to having been influenced by the challenges of the task at hand.
+
+Specification
+==================================
+
+*1. Who determines strategy?*
+
+* A five person/entity board -- Five people is better than three to minimize collusion.
+* Strategic Council should get two-year term so we can pivot people in the middle if necessary. No permanent seats.
+* For the purposes of voting to determine seats (not having seats vote on issues): 1 of the 5 seats should be allocated for miners and signaled through nodes. 1 of the 5 should be ZEC holding weighted so 1 ZEC = 1 vote. 3 of the 5 should be 1 person = 1 vote.
+* Elections should be open such that any person or entity can run for a seat.
+* The board is a paid position from Dev Fund emissions. Compensation TBD.
+
+*2. How do they decide?*
+
+* 20% of block rewards are allocated for the Developer Fund.
+* There should not be any limit up front on where money can go. Perhaps one year it makes sense to invest entirely in protocol and another year it makes sense to invest in user adoption via content marketing, SEO, SEM, etc.
+* Every six months, the board has a responsibility to publish an update to the strategy, key metrics that are being tracked, and key metrics to hit as goals in the next six months. This will require feedback from the community but ultimately the board needs to decide on and own the strategy.
+* Every six months, the board runs a process whereby anyone can submit proposals for how they would best accomplish these strategic objectives and hit those metrics and milestones.
+* No more than 33% of funds can go to one entity for development purposes. This enforces broad decentralization and encourages the ecosystem to identify new participants.
+
+*How are they held accountable for having the correct strategy?*
+
+* Elections every two years from the community.
```

Comment:
The election mechanism must be concretely specified for this proposal to be meaningful and actionable.